src/dot/commons/pose/head_pose.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+import cv2
+import mediapipe as mp
+import numpy as np
+
+mp_face_mesh = mp.solutions.face_mesh
+face_mesh = mp_face_mesh.FaceMesh(
+    min_detection_confidence=0.5, min_tracking_confidence=0.5
+)
+mp_drawing = mp.solutions.drawing_utils
+
+# https://github.com/tensorflow/tfjs-models/blob/838611c02f51159afdd77469ce67f0e26b7bbb23/face-landmarks-detection/src/mediapipe-facemesh/keypoints.ts
```

Comment:
is this link correct? I see no relation to `HEAD_POSE_LANDMARKS`.

Reply:
@AjinkyaIndulkar could you have a look here? https://github.com/google/mediapipe/issues/1615
it doesn't seem to be well documented

Reply:
can you add this link instead?